xlators/cluster/afr/src/afr.c

```diff
@@ -1121,14 +1129,15 @@ struct volume_options options[] = {
         .type = GF_OPTION_TYPE_INT,
         .min = 1,
         .max = INT_MAX,
-        .default_value = 0,
+        .default_value = "0",
```

Comment:
Is this valid? 0 will be out of the range. In previous value, it would be not set (or would be set to NULL). Can someone aware of the option help here?

Reply:
According to the comment in `afr.h`, 0 is valid and means `to turn off quorum enforcement`. If this is so, this should be mentioned in the option help text as well.